### PR TITLE
2.5.0 preliminary changelog & main license update

### PR DESCRIPTION
**Pull request by Bob-Murphy**
## What type of PR is this.
1. [x] Changelog update

********************************************************
Notes:

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,104 @@
...
+  - added ACRE2 items to be given/distributed correctly
+  - added dynamic crewing for rebel garrison statics
+  - added full TFAR BETA compatibility
+  - Convoy mission rework
+  - expanded starting gear for VN
+  - added VN weapon category filtering
+  - Cam Lao Nam map update
+  - Dressup Simulator - added uniforms, headgear, glasses to templates
+  - updated Simplified Chinese translation
+  - fully localised garage hints
```
Comment (CalebSerafin) [inline changelog.txt:21]: Did we add many translations for the garage text, or did add localisation keys for all of the garage text?

Comment (Bob-Murphy) [reply changelog.txt:21]: keys are added, no translations. I'll clarify that.

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,104 @@
+Version 2.5.0 - xxth August 2021
+
+Changes from version 2.4.1 to 2.5.0 Antistasi Community Edition
+
+================ Major ================
+  - implemented new Garage - explanation
+  - implemented Street Artist Navigation Grid Editor (tool for map porting) and changed all systems to work with new navGrids
+  - fully implemented the SOG Prairie Fire (VN) release into the main repo
+  - added new faction templates for 3CB mods
+
```
Comment (HakonRydland) [inline changelog.txt:10]: should mention the license of the new garage and SA

**Review by HakonRydland — CHANGES_REQUESTED** [review]

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,104 @@
...
+  - added ACRE2 items to be given/distributed correctly
+  - added dynamic crewing for rebel garrison statics
+  - added full TFAR BETA compatibility
+  - Convoy mission rework
+  - expanded starting gear for VN
+  - added VN weapon category filtering
+  - Cam Lao Nam map update
+  - Dressup Simulator - added uniforms, headgear, glasses to templates
+  - updated Simplified Chinese translation
+  - fully localised garage hints (no translations added yet)
```
[the remark this answers is not in the document]
Comment (Bob-Murphy) [reply changelog.txt]: done

File: changelog.txt (hunk 2 above)
Comment (Bob-Murphy) [reply changelog.txt:10]: done

**Review by HakonRydland — APPROVED** [review]

**Review by CalebSerafin — APPROVED** [review]
Looks good

**Pull request merged**